conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/ConjureJavaJerseyFeature.java

```diff
@@ -61,9 +62,6 @@ public boolean configure(FeatureContext context) {
         // Tracing
         context.register(new TraceEnrichingFilter());
 
-        // Limits
-        context.register(new RequestLimitingInterceptor());
```

Comment:
where does `RequestLimitingInterceptor`  live now?

Reply:
will be moved into the internal wc product - not a conjure concern